CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 ## Unreleased
 
+## 1.197.1 (Nov, 14 2024)
+ENHANCEMENTS:
+* resource/spotinst_ocean_right_sizing_rule: Corrected format for readability.
```

Comment:
modify to Fixed document format for readability

Reply:
updated